tests/cypress/support/widgets/JupyterLiteSession.ts

```diff
@@ -2,32 +2,15 @@ import { IframeBrowser } from "@mat3ra/tede/src/js/cypress/Browser";
 
 import Widget from "./Widget";
 
-const menuTabSelectorMap: Record<string, string> = {
-    File: "#jp-MainMenu ul li:nth-child(1) div:nth-child(2)",
-    Edit: "#jp-MainMenu ul li:nth-child(2) div:nth-child(2)",
-    View: "#jp-MainMenu ul li:nth-child(3) div:nth-child(2)",
-    Run: "#jp-MainMenu ul li:nth-child(4) div:nth-child(2)",
-    Kernel: "#jp-MainMenu ul li:nth-child(5) div:nth-child(2)",
-    Tabs: "#jp-MainMenu ul li:nth-child(6) div:nth-child(2)",
-    Settings: "#jp-MainMenu ul li:nth-child(7) div:nth-child(2)",
-    Help: "#jp-MainMenu ul li:nth-child(8) div:nth-child(2)",
-};
-
-const menuItemSelectorMap: Record<string, string> = {
-    "Run All Cells": `#jp-mainmenu-run ul li:nth-child(12)`,
-    "Restart Kernel": "#jp-mainmenu-kernel ul li:nth-child(3)",
-    "Restart Kernel and Clear All Outputs": "#jp-mainmenu-kernel ul li:nth-child(4)",
```

Comment:
This was generalized for future implementations

Reply:
We don't have to write a selector for every button, especially when they do not have a convenient class/id attribute. Instead, we can write like this:
```ts
const singleSelector = "#jp-mainmenu-run ul";
this.iframeAnchor.clickOnText("Run All Cells", singleSelector);
this.iframeAnchor.clickOnText("Restart Kernel", singleSelector);
this.iframeAnchor.clickOnText("Restart Kernel and Clear All Outputs", singleSelector);
```